lib/modules.nix

```diff
@@ -134,11 +134,6 @@ let
             ${if prefix == []
               then null  # unset => visible
               else "internal"} = true;
-            # TODO: hidden during the markdown transition to not expose downstream
-            # users of the docs infra to markdown if they're not ready for it.
-            # we don't make this visible conditionally because it can impact
-            # performance (https://github.com/NixOS/nixpkgs/pull/208407#issuecomment-1368246192)
-            visible = false;
```

Comment:
Home Manager is [still not ready](https://github.com/nix-community/home-manager/issues/4142) to properly render Markdown.

Reply:
I'd also much prefer that, but I'm not sure exactly what needs to happen and I don't have the energy to figure it out. You made suggestions in https://github.com/nix-community/home-manager/issues/3680#issuecomment-1441034340 , do you want to try to open a MR at https://gitlab.com/rycee/nmd/ so that we can check if it resolves the breakage, now that we have a [reproducer](https://github.com/nix-community/home-manager/issues/4142#issuecomment-1604044156)?

Reply:
Looks like it's been taken care of :)